client/components/web-preview/index.jsx

```diff
@@ -138,7 +138,7 @@ export class WebPreview extends Component {
 			return;
 		}
 
-		if ( ! this.iframe ) {
+		if ( ! this.iframe || ! this.state.iframeUrl ) {
```

Comment:
Great point, I must have been running on a stale state while testing this out one last time.  Too many devices/browsers open at once.